frontend/sw360-portlet/src/main/resources/META-INF/resources/html/admin/vendors/mergeVendor.jsp

```diff
@@ -166,7 +166,10 @@
                     url: "<liferay-ui:message key="datatables.lang" />"
                 },
             });
-            datatables.enableCheckboxForSelection(table, 0);
+
+            $("#vendorSourcesTable").on( 'init.dt', function (e, settings ) {
```

Comment:
May be the additional parameters `e` and `settings` can be removed as it is not being used currently.

Reply:
Removed additional parameters.